frontend/src/pages/question-service/QuestionPage.js

```diff
@@ -53,7 +53,7 @@ function QuestionPage() {
 
       <div id="questionTagContainer" className="row">
         <div className="questionTag">
-          {questionData.category}
+          {questionData.category.join(", ")}
```

Comment:
I think you can add this here as well next time 😄
```suggestion
        {questionData.category.map((category, index) => (
          <div key={index} className="questionTag">
            {category.trim()}
          </div>
        ))}
```